### DISCUSSION TITLE
能不能选点靠谱的节点

### DISCUSSION THREAD

**Discussion by ktv1978** [getlantern/lantern-forum-cn#966] [Q&A]
现在就US节点靠谱点能使用，其他节点经常打不开网页，甚至连lantern论坛都打不开，现在用lantern上网经常要花费大量时间反复刷新和等待，完全不是一个付费VPN应有的服务。

**Comment by MaxMKMax** [comment]
@ktv1978 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
